pkg/plugins/assign/assign.go

```diff
@@ -84,6 +86,19 @@ func handleGenericComment(pc plugins.Agent, e github.GenericCommentEvent) error
 	return err
 }
 
+func isTeamLogin(login string) bool {
```

Comment:
IMHO the following is a better approach:
```go
func isTeamLogin(login string) bool {
	match, err := regexp.MatchString(`^[a-zA-Z0-9]+/[a-zA-Z0-9]+$`, login)
	if err != nil {
		return false
	}
	return match
}

```

Reply:
Thank you!!

fixed it https://github.com/kubernetes-sigs/prow/pull/164/commits/e04afd17ba3b2f5e0bc114d3eaa314b69d80538a

Reply:
I made a few more corrections, this is because it didn’t work well with teams or organizations that include a “-”.

https://github.com/kubernetes-sigs/prow/pull/164/commits/4af6ef4f7ec331aa223947220fc83732419df45c